### PR TITLE
added error message when passwords not the same

### PR DESCRIPTION
**Pull request by bryanyuan1**
When the second password differ from the first one, it would show an error message.
- the confirm password checks ONLY if it is the same with the first one

### CODE REVIEW

File: client/src/components/General/Signup/Component.jsx [hunk 1]
```diff
@@ -116,12 +119,13 @@ class Signup extends Component {
...
                           name="confirmPassword"
                           type="password"
+                          label="Confirmation"
                           // Note: the order of validators matter
-                          validate={composeValidators(
-                            passwordRequired,
-                            passwordLength,
-                            passwordFormat
-                          )}
+                          validate={confirmation({
```
Comment (djuanit0x) [inline client/src/components/General/Signup/Component.jsx]: I think we should move this validation to `client/src/utils/validationRules.js`

File: client/src/components/General/Signup/Component.jsx [hunk 2]
```diff
@@ -20,7 +21,8 @@ import {
   emailRequired,
   passwordFormat,
   passwordLength,
-  passwordRequired
+  passwordRequired,
+  passwordConfirmed
```
Comment (djuanit0x) [inline client/src/components/General/Signup/Component.jsx]: you import `passwordConfirmed` but there is no inside `client/src/utils/validationRules.js`

File: client/src/components/General/Signup/Component.jsx (hunk 1 above)
Comment (bryanyuan1) [reply client/src/components/General/Signup/Component.jsx]: Moved function to validationRules.js

File: client/src/components/General/Signup/Component.jsx (hunk 2 above)
Comment (bryanyuan1) [reply client/src/components/General/Signup/Component.jsx]: fixed.

**Pull request merged**